utopia-remix/app/routes-test/login.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { newTestRequest } from '../test-util'
+import { handleLogin } from '../routes/login'
+import { ServerEnvironment } from '../env.server'
+
+describe('handleLogin', () => {
+  let corsOrigin: string
+  beforeAll(() => {
+    corsOrigin = ServerEnvironment.CORS_ORIGIN
+    // this is since in the tests we set CORS_ORIGIN to '*',
```

Comment:
note: this is actually a leftover bug, we should have it already as `localhost:8000` (maybe it's worth doing a mini PR for this one!) (and it should not be named `CORS_ORIGIN` anymore but `REMIX_URL` or something like that)